### PR TITLE
coincurve for cosmos ledger plugin

### PR DESCRIPTION
**Pull request by solarw**
## Proposed changes

coincurve for cosmos ledger plugin

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes and CI passes too
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by marcofavorito — APPROVED** [review]
LGTM

File: Pipfile [hunk 1]
```diff
@@ -20,6 +20,7 @@ colorlog = "==4.1.0"
 defusedxml = "==0.6.0"
 docker = "==4.2.0"
 ecdsa = ">=0.15"
+coincurve = ">=15.0.0"
```
Comment (marcofavorito) [inline Pipfile:23]: why here, if `coincurve` is in `setup.py`?

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]
This currently does not play nicely with the ACN. We need to investigate why and resolve this before continuing with it.

**Comment by DavidMinarsch** [comment]
Outdated - closing now

**Pull request closed without merging**